src/client/pages/OfferNew/Introduction/Sidebar/DetailsModal/utils.ts

```diff
@@ -600,7 +632,20 @@ const getInitialNorwegianTravelValues = (
   },
 })
 
-export const getInitialInputValues = (offerQuote: OfferQuote) =>
+export const getInitialInputValues = (
+  person: OfferPersonInfo,
+  offerQuote: OfferQuote,
+) => {
+  const { firstName, lastName, birthDate } = person
```

Comment:
Should we maybe name this something like `personalInfo` or `memberDetails` ? Person sounds a bit weird to me :)

Reply:
I agree! The reason i chose person was that the property of OfferData is named "person". That property is used in several files. Might be better to change that as well?

Reply:
aha! then let's not bother now 🙌